src/content/en/fundamentals/primers/service-workers/a-basic-service-worker.markdown

```diff
@@ -0,0 +1,183 @@
+  shows you how a client gets, or registers, a service worker. It shows you how 
+  a service worker prepares to act as a proxy for its clients. Both subjects 
+  have more depth than is shown in this section. It's not intended to be 
+  comprehensive; It's intended to give you a basic foundation on which to build 
+service worker knowledge."
+authors:
+- josephmedley
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 4
+key-takeaways:
+  tldr:   
+  - "Call <code>register()</code> to install a service worker." 
+  - "Learn to use Promises before attempting service workers."
+  - "Service workers have a limited scope and requrie HTTPS."
+  - "The install process has two events: install and activate."
+  - "Use <code>claim()</code> to start proxying immediately."
+notes:
+  promises:
+    - "<b>Promises</b>&mdash;Notice the use of <code>.then()</code> at the end of the <code>register()</code> function. This is an example of an ECMAScript 2015 construct called a <a href='https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/Promise.jsm/Promise'>Promise</a>. Service workers make heavy use of Promises. If you've never used Promises before, you should familiarize yourself with them before trying to implement a service worker."
+  https-only:
+    - "<b>HTTPS Only</b>&mdash;As we'll see later, service workers can do almost whatever they want to HTTP requests and responses. Since this would make them targets for man-in-the-middle attacks, they must be served over HTTPS.This doesn't mean you need HTTPS for development and testing. Service workers served over localhost will also work."
+  sws-dont-control:
+    - "<b>Service Workers Don't Take Control</b>&mdash;If you look around the web, you'll find that many of the pages discussing service workers refer to the service worker as 'taking control' of a page. But as we saw earlier, a service workers can't do anything to its clients. That's why it may be more accurate to think of a service worker as ready to proxy."
+---
+
+<p class="intro">
+  This section describes the most basic service worker possible. It shows you 
+  how a client gets, or registers, a service worker. It shows you how a service 
+  worker prepares to act as a proxy for its clients. Both subjects have more 
+  depth than is shown in this section. It's not intended to be comprehensive; 
+  It's intended to give you a basic foundation on which to build service worker 
+  knowledge.</p>
+
+{% include shared/toc.liquid %}
+
+{% include shared/takeaway.liquid list=page.key-takeaways.tldr %}
+
+## A client registers a service worker
+
+To use a service worker, a client tells the browser to install it by calling 
+`register()`. Every page that needs the service worker must implement register(), 
+but only the first page to do so will trigger the download. The most basic 
+register implementation looks like this:
+
+{% highlight javascript %}
+//Does the browser support service workers?
```

Comment:
Always include spaces after `//`, you're inconsistent, in some cases you've included spaces and some not. A space generally makes things look nicer.